content/references/http-websocket-apis/public-api-methods/server-info-methods/nft_info.md

```diff
@@ -0,0 +1,134 @@
+---
+html: nft_info.html
+parent: server-info-methods.html
+blurb: Retrieve status of the specified NFT in human-readable format.
+labels:
+  - Non-fungible Tokens, NFTs
+---
+# nft_info
+[[Source]](https://github.com/XRPLF/clio/blob/4a5cb962b6971872d150777881801ce27ae9ed1a/src/rpc/handlers/NFTInfo.cpp "Source")
+
+The `nft_info` command asks the Clio server for a human-readable version of information about [the NFT](non-fungible-tokens.html) being queried. 
+
+## Request Format
+An example of the request format:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+{
+  "id": 1,
+  "command": "nft_info",
+  "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C"
+}
+```
+
+*JSON-RPC*
+
+```json
+{
+    "method": "nft_info",
+    "params": [
+      {
+          "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C"
+      }
+    ]
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+[Try it! >](websocket-api-tool.html#nft_info)
+
+The request contains the following parameters:
+
+| `Field`        | Type                       | Description                    |
+|:---------------|:---------------------------|:-------------------------------|
+| `nft_id`       | String                     | A unique identifier for the non-fungible token (NFT). |
+| `ledger_hash`  | String                     | _(Optional)_ A 20-byte hex string for the ledger version to use. If you do not specify this field, it defaults to `validated`. (See [Specifying Ledgers][]) |
+| `ledger_index` | String or Unsigned Integer | _(Optional)_ The [ledger index][] of the ledger to use, or a shortcut string to choose a ledger automatically. If you do not specify this field, it defaults to `validated`. Do not specify the `ledger_index` as `closed` or `current`. Specifying `current` forwards the request to the P2P `rippled` server and the `nft_info` API is not available on `rippled`. (See [Specifying Ledgers][]) | 
+
+## Response Format
+
+An example of a successful response:
+
+<!-- MULTICODE_BLOCK_START -->
+
+*WebSocket*
+
+```json
+  "result": {
+    "nft_id": "0000000025CC40A6A240DB42512BA22826B903A785EE2FA512C5D5A70000000C",
+    "ledger_index": 2436210,
+    "owner": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "is_burned": false,
+    "flags": 0,
+    "transfer_fee": 0,
+    "issuer": "rhSigFwZ9UnbiKbpaco8aSQUsNFXJVz51W",
+    "nft_taxon": 0,
+    "nft_sequence": 12,
+    "uri": "ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf4dfuylqabf3oclgtqy55fbzdi",
+    "validated": true,
+    "status": "success",
+    "warnings": [
+      "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request",
+      "This server may be out of date"
+    ]
```

Comment:
`warnings` is up one level in the next release. IMO we should just document that since it will be released soon. trying to determine when that will be. here's an example that we can use if so:


```
{
  "result": {
    "nft_id": "00080000B4F4AFC5FBCBD76873F18006173D2193467D3EE70000099B00000000",
    "ledger_index": 269,
    "owner": "rG9gdNygQ6npA9JvDFWBoeXbiUcTYJnEnk",
    "is_burned": false,
    "flags": 8,
    "transfer_fee": 0,
    "issuer": "rHVokeuSnjPjz718qdb47bGXBBHNMP3KDQ",
    "nft_taxon": 0,
    "nft_sequence": 0,
    "uri": "https://xrpl.org",
    "validated": true,
    "status": "success"
  },
  "warnings": [
    {
      "id": 2001,
      "message": "This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request"
    },
    {
      "id": 2002,
      "message": "This server may be out of date"
    }
  ]
}
```

Reply:
Caution on predicting the future - we should document how things work when the doc is published. If there's going to be an imminent change, I suggest holding the publication until the feature is live. Otherwise, it's best to document reality today and update it when the change is live.

Reply:
+1 to that. I propose handling this in a separate PR that can be merged once the next release is live.

Reply:
The release is imminent, the code has already been merged to develop. Please update the documentation to what is in develop

Reply:
We can hold this documentation until the release itself is live, which will be very soon

Reply:
https://github.com/XRPLF/xrpl-dev-portal/pull/1462 has been created to capture the imminent change.